File: 1440-convert-integer-to-the-sum-of-two-no-zero-integers/1440-convert-integer-to-the-sum-of-two-no-zero-integers.cpp
class Solution {
public:
    vector<int> getNoZeroIntegers(int n) {
        for(int i = 1; i <= n / 2; i++)
        {
            if(!containsZero(i) && !containsZero(n - i))
            {
                return {i, n - i};
            }
        }
        return {};
    }
private:
    bool containsZero(int x)
    {
        while(x > 0)
        {
            if(x % 10 == 0)
            {
                return true;
            }
            x /= 10;
        }
        return false;
    }
};

File: 1440-convert-integer-to-the-sum-of-two-no-zero-integers/1440-convert-integer-to-the-sum-of-two-no-zero-integers.py
class Solution:
    def containsZero(self, x: int) -> int:
        while x > 0:
            if x % 10 == 0:
                return True
            x //= 10
        return False 
    def getNoZeroIntegers(self, n: int) -> List[int]:
        for i in range(1, n // 2 + 1):
            if not self.containsZero(i) and not self.containsZero(n - i):
                return [i, n - i]
        return []
